__tests__/pages/resource/id/careGaps.test.tsx

```diff
@@ -0,0 +1,193 @@
+import { render, screen, act, fireEvent, within } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import {
+  createMockRouter,
+  getMockFetchImplementation,
+  mockResizeObserver,
+} from "../../../helpers/testHelpers";
+import { RouterContext } from "next/dist/shared/lib/router-context";
+import CareGapsPage from "../../../../pages/[resourceType]/[id]/careGaps";
+import { DateTime } from "luxon";
+import { fhirJson } from "@fhir-typescript/r4-core";
+
+const MEASURE_BODY_WITH_DATES = {
+  resourceType: "Measure",
+  id: "measure-EXM104-8.2.000",
+  meta: {
+    lastUpdated: "2022-07-21T18:12:25.008Z",
+  },
+  effectivePeriod: {
+    start: "2019-01-01",
+    end: "2019-12-31",
+  },
+};
+
+const RESOURCE_ID_BODY: fhirJson.Bundle = {
+  resourceType: "Bundle",
+  meta: {
+    lastUpdated: "2022-06-23T19:52:58.721Z",
+  },
+  type: "searchset",
+  total: 2,
+  entry: [
+    {
+      fullUrl: "http://localhost:3000/4_0_1/PractitionerReport/denom-EXM125-3",
+      resource: {
+        resourceType: "Practitioner",
+        id: "denom-EXM125-3",
+        meta: {
+          profile: [
+            "http://hl7.org/fhir/us/core/StructureDefinition/us-core-practitionerreport-note",
+          ],
+        },
+      },
+    },
+    {
+      fullUrl: "http://localhost:3000/4_0_1/PractitionerReport/numer-EXM125-3",
+      resource: {
+        resourceType: "Practitioner",
+        id: "numer-EXM125-3",
+        meta: {
+          profile: [
+            "http://hl7.org/fhir/us/core/StructureDefinition/us-core-practitionerreport-note",
+          ],
+        },
+      },
+    },
+  ],
+};
```

Comment:
I have seen that this is in a couple of test files and it is pretty long so @mgramigna would it be okay/beneficial for them to pull this out into it's own file?

Reply:
no preference, fine either way